tests/test_auxiliary_functions.py

```diff
@@ -0,0 +1,20 @@
+"""
+Tests for polaraveraging.py
+"""
+import clmm
+import clmm.polaraveraging as pa
+from numpy import testing
+import numpy as np
+
+
+def test_theta_units_conversion():
+    # tests for invaid input
+    testing.assert_raises(ValueError, pa._theta_units_conversion, np.pi, 'crazy units')
+    testing.assert_raises(ValueError, pa._theta_units_conversion, np.pi, 'Mpc')
```

Comment:
Lets move physical units and angular units conversions to diff functions. Split up tests as well